ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -5,22 +5,25 @@
 #
 
 - name: create group
+  when: not os|startswith("zos")
   group: name="{{ server_user }}"
 
 - name: create user
+  when: not os|startswith("zos")
   user:
     name: "{{ server_user }}"
     group: "{{ server_user }}"
     shell: "{{ bash_path[os|stripversion]|default('/bin/bash') }}"
 
 - name: add ::1 to /etc/hosts for ipv6 compat
+  when: not os|startswith("zos")
   lineinfile:
     dest: /etc/hosts
     state: present
     line: ::1 localhost.localdomain localhost
 
 - name: prepare installing tap2junit
-  when: type != "release"
+  when: type != "release" and not os|startswith("zos")
```

Comment:
Perhaps I'm missing something, but there's a partial added for this for zos. So, is it added but not run for some reason or run somewhere else?

Reply:
I think I added it hoping it could install tap2junit, but did not manage to get it working.  I've removed the partial for now, and will revisit when we come back to get the parts needed for build Node.js working (right now I've focused on what was needed to build libuv)

Reply:
Maybe put a `#TODO(mhdawson): get tap2junit working on zOS` for future reference (so we know it's outstanding).